specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json

```diff
@@ -3637,6 +4158,169 @@
+          "enum": [
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ArchivePackageSourceProperties": {
+      "description": "The properties of the archive package source.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of package source for a archive.",
+          "type": "string"
+        },
+        "url": {
+          "description": "The external repository url.",
+          "type": "string"
+        },
+        "distributionName": {
+          "description": "Upstream Debian/RPM distribution Name.",
+          "type": "string"
+        }
+      }
+    },
+    "ArchiveUpdateParameters": {
+      "description": "The parameters for updating a archive.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ArchiveUpdateProperties",
+          "description": "The properties of the connected registry update parameters.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ArchiveUpdateProperties": {
+      "description": "The properties of a archive.",
+      "type": "object",
+      "properties": {
+        "publishedVersion": {
+          "description": "The published version of the archive.",
+          "type": "string"
+        }
+      }
+    },
+    "ArchiveVersionListResult": {
+      "description": "The result of a request to list export pipelines for a container registry.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of export pipelines. Since this list may be incomplete, the nextLink field should be used to request the next list of export pipelines.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ArchiveVersion"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next list of pipeline runs.",
+          "type": "string"
+        }
+      }
+    },
+    "ArchiveVersion": {
+      "description": "An object that represents an export pipeline for a container registry.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ArchiveVersionProperties",
+          "description": "The properties of the archive.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ArchiveVersionProperties": {
+      "description": "The properties of an export pipeline.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```

Comment:
>provisioningState [](http://example.com/codeflow?start=9&length=17)

duplicated with that in "ArchiveProperties". please consider define a separate "ProvisioningState" definition